lib/active_fulfillment/services/amazon_mws.rb

```diff
@@ -122,23 +84,37 @@ def endpoint
 
     def fulfill(order_id, shipping_address, line_items, options = {})
       requires!(options, :order_date, :shipping_method)
-      commit :post, :outbound, :create, build_fulfillment_request(order_id, shipping_address, line_items, options)
+      with_error_handling do
+        data = commit :post, 'CreateFulfillmentOrder', build_fulfillment_request(order_id, shipping_address, line_items, options)
+        parse_fulfillment_response(parse_document(data), 'Successfully submitted the order')
+      end
     end
 
     def status
-      commit :post, :outbound, :status, build_status_request
+      with_error_handling do
+        data = commit :post, 'GetServiceStatus', build_basic_api_query({ :Action => 'GetServiceStatus' })
+        parse_tracking_response(parse_document(data))
+      end
```

Comment:
I think this will swallow ActiveUtils exceptions, which means that this method will return, even when there's network issues. Is this intentional? 

In this cases I also prefer just adding a `rescue` block to the method instead of the helper method but YMMV.